docs/rst/notes/forthcoming_version.rst

```diff
@@ -6,7 +6,3 @@
 ###################
```

Comment:
tag this file as orphan (see [this PR](https://github.com/eProsima/DDS-Router/pull/442/files) as referece)